discovery_engine_core/core/src/engine.rs

```diff
@@ -53,8 +53,27 @@ pub enum Error {
     Errors(Vec<Error>),
 }
 
+#[allow(dead_code)]
+/// Feed market.
+struct Market {
+    country_code: String,
+    lang_code: String,
+}
+
+#[allow(dead_code)]
+/// Discovery Engine configuration settings.
+pub struct Config {
+    api_key: String,
+    api_base_url: String,
+    markets: Vec<Market>,
+    smbert_vocab: Vec<u8>,
+    smbert_model: Vec<u8>,
```

Comment:
here we also need the kpe assets (vocab, model, cnn, classifer)
https://github.com/xaynetwork/xayn_discovery_engine/pull/93#issuecomment-1026717249